lyric_finder/src/lib.rs

```diff
@@ -136,6 +138,45 @@ impl Default for Client {
     }
 }
 
+fn improve_query(query: &str) -> String {
```

Comment:
can you add a function document explaining the function's purpose? Maybe, "Improve the query by filtering out redundant parts, e.g `Remix`, `Remsatered` that may result in incorrect search results"?